astropy/io/votable/tests/test_table.py

```diff
@@ -261,6 +263,82 @@ def test_write_with_format():
     assert b"TABLEDATA" not in obuff
 
 
+@pytest.mark.skipif(not HAS_PYARROW, reason="requires pyarrow")
+@pytest.mark.parametrize("overwrite", [True, False])
+def test_read_write_votable_parquet(tmp_path, overwrite):
+    """
+    Test to write and read VOTable with Parquet serialization
+    """
+
+    # Create some fake data
+    number_of_objects = 10
+    ids = [f"COSMOS_{ii:03g}" for ii in range(number_of_objects)]
+    redshift = np.random.uniform(low=0, high=3, size=number_of_objects)
+    mass = np.random.uniform(low=1e8, high=1e10, size=number_of_objects)
+    sfr = np.random.uniform(low=1, high=100, size=number_of_objects)
+    astropytab = Table([ids, redshift, mass, sfr], names=["id", "z", "mass", "sfr"])
+
+    # Create Column metadata
+    column_metadata = {
+        "id": {"unit": "", "ucd": "meta.id", "utype": "none"},
+        "z": {"unit": "", "ucd": "src.redshift", "utype": "none"},
+        "mass": {"unit": "solMass", "ucd": "phys.mass", "utype": "none"},
+        "sfr": {"unit": "solMass / yr", "ucd": "phys.SFR", "utype": "none"},
+    }
+
+    # Write VOTable with Parquet serialization
+    filename = tmp_path / "test_votable_parquet.vot"
+    astropytab.write(
+        filename,
+        column_metadata=column_metadata,
+        overwrite=overwrite,
+        format="votable.parquet",
+    )
+
+    # Check both files are written out
+    assert set(os.listdir(tmp_path)) == {
```

Comment:
I think tmp_path is a Path object, so you might be able to use pathlib methods directly (unconfirmed): https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module